assembly/runtime.ts

```diff
@@ -726,45 +735,47 @@ class GrpcCallback {
  */
 export class RootContext extends BaseContext {
   // hack to workaround lack of OOP
-  validateConfiguration_: (thiz: RootContext, configuration_size: size_t) => bool;
-  onConfigure_: (thiz: RootContext, configuration_size: size_t) => bool;
-  onStart_: (thiz: RootContext, vm_configuration_size: size_t) => bool;
+  createContext_: (thiz: RootContext, context_id: u32) => Context;
+  validateConfiguration_: (thiz: RootContext, configuration_size: usize) => bool;
+  onConfigure_: (thiz: RootContext, configuration_size: usize) => bool;
+  onStart_: (thiz: RootContext, vm_configuration_size: usize) => bool;
   onTick_: (thiz: RootContext) => void;
   done_: (thiz: RootContext) => void;
-  createContext_: (thiz: RootContext) => Context;
   onQueueReady_: (thiz: RootContext, token: u32) => void;
-
-  private http_calls_: Map<u32, HttpCallback>;
-  private grpc_calls_: Map<u32, GrpcCallback>;
-
-  constructor() {
-    super();
-    this.http_calls_ = new Map();
-    this.grpc_calls_ = new Map();
-    this.validateConfiguration_ = (thiz: RootContext, configuration_size: size_t) => { return thiz.validateConfiguration(configuration_size); };
-    this.onConfigure_ = (thiz: RootContext, configuration_size: size_t) => { return thiz.onConfigure(); };
-    this.onStart_ = (thiz: RootContext, vm_configuration_size: size_t) => { return thiz.onStart(vm_configuration_size); };
+  onHttpCallResponse_: (thiz: RootContext, token: u32, headers: u32, body_size: u32, trailers: u32) => void;
+
+  private configuration_: string = "default_configuration"
```

Comment:
Added configuration here, since it seems like this can be a default solution to get envoy filter configs.

Reply:
why not default it to an empty string?

Reply:
Good point. I will change that.